TombEngine/Objects/TR4/Entity/tr4_demigod.cpp

```diff
@@ -171,6 +171,7 @@ namespace TEN::Entities::TR4
 
 			fx->pos.Orientation.z = 0;
 
+			fx->fallspeed = 0;
```

Comment:
Does this value (gravity) also provokes incorrect bubble behaviour?

Reply:
removed